packages/maskbook/src/plugins/RedPacket/SNSAdaptor/RedPacketDialog.tsx

```diff
@@ -233,7 +233,7 @@ export default function RedPacketDialog(props: RedPacketDialogProps) {
             },
             {
                 label: t('plugin_red_packet_select_existing'),
-                children: <RedPacketHistoryList onSelect={onCreateOrSelect} onClose={onClose} />,
+                children: <RedPacketPast onSelect={onCreateOrSelect} onClose={onClose} />,
```

Comment:
When I'm at 'Past-Collectibles' tab, then I click 'New' tab, it will jump to 'Token' tab which is not desirable.

Reply:
May refactor this nested tabs to some route-like later.